components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -545,6 +607,11 @@ void RRTMGPRadiation::run_impl (const double dt) {
   auto ts = timestamp();
   auto update_rad = scream::rrtmgp::radiation_do(m_rad_freq_in_steps, ts.get_num_steps());
 
+  if (m_time_varying_active_gases_file[0] != "nofilegiven") {
```

Comment:
For simplicity of the code you could make a single bool variable that you initialize in the init_impl and use that for all of these logicals.  Maybe something like
```
m_time_vary_active_gas_from_file = m_time_varying_active_gases_file[0] != "nofilegiven" ? true : false;
```